### PR TITLE
style: the final comments refactor.

### PR DESCRIPTION
**Pull request by Jaskowicz1**
This PR is the final refactor to comments (continuation from #979, part 4)! Now we can relax knowing our comments follow the coding style guide. 

## Code change checklist

- [x] I have ensured that all methods and functions are **fully documented** using doxygen style comments.
- [x] My code follows the [coding style guide](https://dpp.dev/coding-standards.html).
- [x] I tested that my change works before raising the PR.
- [x] I have ensured that I did not break any existing API calls.
- [x] I have not built my pull request using AI, a static analysis tool or similar without any human oversight.

### CODE REVIEW

File: include/dpp/unicode_emoji.h [hunk 1]
```diff
@@ -3,9 +3,11 @@
 namespace dpp {
 
 /**
- * The unicode emojis in this namespace are auto-generated from https://raw.githubusercontent.com/ArkinSolomon/discord-emoji-converter/master/emojis.json
+ * @brief Emoji unicodes.
  *
- * If you want to use this, you have to pull the header in separately. e.g.
+ * @note The unicode emojis in this namespace are auto-generated from https://raw.githubusercontent.com/ArkinSolomon/discord-emoji-converter/master/emojis.json
```
Comment (braindigitalis) [inline include/dpp/unicode_emoji.h:8]: you edited the note, telling you not to hand edit the file 😁 
if you want to change this file you have to change the php file that generates it under buildtools. be sure to test it!

File: include/dpp/voiceregion.h [hunk 2]
```diff
@@ -31,10 +31,10 @@ namespace dpp {
  * @brief Flags related to a voice region
  */
 enum voiceregion_flags {
-	v_optimal	= 0x00000001,
-	v_deprecated	= 0x00000010,
-	v_custom	= 0x00000100,
-	v_vip		= 0x00001000
+	v_optimal = 0x00000001,
```
Comment (braindigitalis) [inline include/dpp/voiceregion.h:34]: while youre in here perhaps we can add a comment on each of these for what they do?

**Review by braindigitalis — CHANGES_REQUESTED** [review]

File: include/dpp/unicode_emoji.h (hunk 1 above)
Comment (Jaskowicz1) [reply include/dpp/unicode_emoji.h:8]: Ahhh I thought the php was just editing the content below 😅

File: include/dpp/voiceregion.h (hunk 2 above)
Comment (Jaskowicz1) [reply include/dpp/voiceregion.h:37]: Yep! All done!

**Review by braindigitalis — APPROVED** [review]

**Pull request merged**